client/post-editor/editor-ground-control/index.jsx

```diff
@@ -399,6 +479,17 @@ export default React.createClass( {
 						this.schedulePostPopover()
 					}
 				</div>
+				{
+					this.state.needsVerification &&
+					<div className="editor-ground-control__email-verification-notice" tabIndex={ 7 } onClick={ this.props.onMoreInfoAboutEmailVerify }>
+						<Gridicon
+							icon="info"
+							className="editor-ground-control__email-verification-notice-icon" />
+						{ this.getVerificationNoticeLabel() }
+						{ ' ' }
+						<u>{ this.translate( 'Learn More' ) }</u>
```

Comment:
Are you using `<u>` for an underline effect? That behavior was [deprecated in the HTML 4 specification](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u). The correct semantic element here would be a `<button>`, which is [reset in our styles to remove default styling](https://github.com/Automattic/wp-calypso/blob/0cfd9a6296bdbacf94bed4ec49d805c445c81c56/client/components/button/style.scss#L6-L15).